src/__tests__/components/reset.password.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ResetPasswordComponent from '../../App/Pages/reset.password.page';
+import Data from '../../__mocks__/reset.password.data';
+import config from '../../helper/test.helper';
+
+Enzyme.configure({
+	adapter: new Adapter(),
+});
+
+const component = (
+	<ResetPasswordComponent
+		match={{
+			params: {
+				token: 'ewfwefwe435rfcewf',
+			},
+		}}
+	/>
+);
+let store, wrapper;
+
+describe('Reset Password Page', () => {
+	it('test not started', done => {
+		store = config.mockStore(Data.not_started);
+		wrapper = config.mountNewWrapper(store, component);
+		expect(wrapper.find('div.mainContainer')).toExist();
+		done();
+	});
+	it('test pending', done => {
+		store = config.mockStore(Data.pending);
+		wrapper = config.mountNewWrapper(store, component);
+		expect(wrapper.find('div.mainContainer')).toExist();
+		done();
+	});
+	it('test success', done => {
+		store = config.mockStore(Data.success);
+		wrapper = config.mountNewWrapper(store, component);
+		expect(wrapper.find('div.mainContainer')).toExist();
+		done();
+	});
+	it('test failed', done => {
+		store = config.mockStore(Data.failed);
+		wrapper = config.mountNewWrapper(store, component);
+		expect(wrapper.find('div.mainContainer')).toExist();
+		done();
+	});
+	it('test error', done => {
+		store = config.mockStore(Data.error);
+		wrapper = config.mountNewWrapper(store, component);
+		expect(wrapper.find('div.mainContainer')).toExist();
```

Comment:
the same here, try to test different things to ensure the feature is working properly